src/PrestaShopBundle/Controller/Admin/Configure/ShopParameters/OrderPreferencesController.php

```diff
@@ -27,69 +27,51 @@
 namespace PrestaShopBundle\Controller\Admin\Configure\ShopParameters;
 
 use PrestaShopBundle\Controller\Admin\FrameworkBundleAdminController;
-use PrestaShopBundle\Security\Voter\PageVoter;
-use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use PrestaShopBundle\Security\Annotation\AdminSecurity;
 use Symfony\Component\HttpFoundation\RedirectResponse;
 use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
 
 /**
  * Controller responsible of "Configure > Shop Parameters > Order Settings" page.
  */
 class OrderPreferencesController extends FrameworkBundleAdminController
 {
     /**
-     * Show order preferences page.
+     * @AdminSecurity("is_granted(['read', 'update', 'create', 'delete'], request.get('_legacy_controller'))")
      *
      * @param Request $request
      *
-     * @Template("@PrestaShop/Admin/Configure/ShopParameters/OrderPreferences/order_preferences.html.twig")
-     *
-     * @return array Template parameters
+     * @return Response
      */
     public function indexAction(Request $request)
     {
         $legacyController = $request->attributes->get('_legacy_controller');
 
         $form = $this->get('prestashop.admin.order_preferences.form_handler')->getForm();
 
-        return [
+        return $this->render('@PrestaShop/Admin/Configure/ShopParameters/OrderPreferences/order_preferences.html.twig', [
             'layoutTitle' => $this->trans('Order settings', 'Admin.Navigation.Menu'),
             'requireAddonsSearch' => true,
             'enableSidebar' => true,
             'help_link' => $this->generateSidebarLink($legacyController),
             'form' => $form->createView(),
             'isMultishippingEnabled' => $this->configuration->getBoolean('PS_ALLOW_MULTISHIPPING'),
             'isAtcpShipWrapEnabled' => $this->configuration->getBoolean('PS_ATCP_SHIPWRAP'),
-        ];
+        ]);
     }
 
     /**
-     * Handle order settings form submit.
```

Comment:
why did you remove the PHPDoc, I'm curious :)

Reply:
Because the controller has only 2 actions: `indexAction` and `processAction`. For me, it's obvious that the 1st handles GET calls and the other one handles form submissions, no need to add it in the phpdoc.

It's like
```
<?php

/**
 * Cat object
 */
class Cat
{
}
```